port-chooser/pom.xml

```diff
@@ -56,13 +55,13 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
+      <groupId>org.hamcrest</groupId>
```

Comment:
Based on the documentation, the `hamcrest` and `junit` dependencies should have been ordered hamcrest first.